app/notifications/push/push.ios.ts

```diff
@@ -49,12 +54,9 @@ class PushNotification {
 		NotificationsIOS.setBadgesCount(count);
 	};
 
-	async configure(params) {
-		this.onRegister = params.onRegister;
-		this.onNotification = params.onNotification;
-
+	async configure(onNotification: (notification: INotification) => void) {
+		this.onNotification = onNotification;
 		const initial = await NotificationsIOS.getInitialNotification();
-		// NotificationsIOS.consumeBackgroundQueue();
```

Comment:
why?

Reply:
This code already commented before, as I don't know why they commented, I decided to leave it.